### DISCUSSION TITLE
Set cost for GeoFence over a given timeframe

### DISCUSSION THREAD

**Discussion by digisz** [teslamate-org/teslamate#1570] [Ideas]
I'm in the situation where I only get a twice a year bill for my charging at home. The rate I'm paying depends on multiple factors and hence my cost per kw/h is a guestimate at best.

It would be great to be able to set an overall cost for a defined geofence in a certain date-range.

**Comment by tobiasehlert** [comment]
Hi @digisz,

Have you check the [MattJeanes/TeslaMateAgile](https://github.com/MattJeanes/TeslaMateAgile) project, if it meets your requirements?
There you can specify rates depending on hours of the day and that is maybe something that you can use in your situation.

Regards,
Tobias
